src/handlers/handleIngressAPI.ts

```diff
@@ -22,7 +30,34 @@ async function makeIngressRequest(receivedRequest: Request, env: IntegrationEnv)
   addProxyIntegrationHeaders(request.headers, receivedRequest.url, env)
 
   console.log(`sending ingress request to ${url.toString()}...`)
-  return fetch(request, { backend: 'fpjs' })
+
+  if (!isOpenClientResponseEnabled(env)) {
+    return fetch(request, { backend: 'fpjs' })
+  }
+
+  const secretStore = new SecretStore(FingerprintSecretStoreName)
+  const decryptionKey = await secretStore.get(FingerprintDecryptionKeyName).then((v) => v?.plaintext())
+  if (!decryptionKey) {
+    throw new Error('Decryption key not found in secret store')
+  }
+
+  try {
+    const response = await fetch(request, { backend: 'fpjs' })
+
+    // Parse the open response
+    const text = await response.text()
+    const event = await unsealData(JSON.parse(text).sealedResult, decryptionKey)
+    void processOpenClientResponse({ event, sealedResult: JSON.parse(text).sealedResult }) // void means skip awaiting this and handle it in the background
```

Comment:
Could you explain how `void` helps here? I thought even without `void`, this will not wait until `processOpenClientResponse` is complete, so writing or not writing `void` here doesn't make any difference.

Reply:
Void means that I specifically skipped awaiting this promise, and I'm not going to handle its `catch` or `then` callbacks. `Void` doesn't skips the `promise` but tells IDE that it's deliberately unhandled.

Reply:
Let's discuss and decide on this because I would like not to introduce any changes.

Reply:
I prefer using empty `.finally()` but this also sounds fair. I remember getting `no-void warning` when using `void`. If you don't face the same, then it is OK